apps/remix-ide/src/blockchain/providers/vm.ts

```diff
@@ -1,4 +1,5 @@
-import Web3 from 'web3'
+import Web3, { FMT_BYTES, FMT_NUMBER, LegacySendAsyncProvider } from 'web3'
+import { fromWei } from 'web3-utils-legacy'
```

Comment:
This was a temporary decision made to reduce the friction of migrating to `4.x` since `BN` is no longer used

Reply:
I've removed this dependency